docs/source/examples/checks/performance/simple_model_comparison.ipynb

```diff
@@ -5,118 +5,76 @@
    "id": "840ea662",
```

Comment:
<p>"There are 3 types of simple models exists in the check. " -&gt; grammar still incorrect (did you change it?)</p><p>ok nvm lets just rewrite this:</p><p>The check has three possible "simple model" heuristics, from which one is chosen and compared to. By default the check uses the "Constant" heuristic, which can be overriden in the checks' parameters. There is no simple model which is more "correct" to use, each gives a different baseline to compare to, and you may experiment with the different types and see how it performs on your data.</p><p> </p><p>given that the params are without capslock, probably better also here... </p><p>phrasing suggestion (fixing some grammar also):</p><p>A <strong>constant</strong> model - the default. For regression the prediction is ....</p><p>A <strong>random</strong> model - draws the prediction from the distribution of the labels in the train.</p><p>A <strong>tree</strong> model - trains a simple decision tree with a given depth.</p><p> </p><p>(how can the tree depth be defined?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>